### PR TITLE
Feature: Update to 0.8.0b

### PR DESCRIPTION
**Pull request by daniel-wtd**
- update: screenshots
- update: jdk11 to 19.08
- update: SPD to 0.8.0b

---

- fixes: #6
- fixes: #9
- see also: #8

### CODE REVIEW

**Comment by daniel-wtd** [comment]
@bilelmoussaoui The build currently fails due to wrong metadata_license in appdata.yml

Afaik, it should be valid, according to the spdx code.

**Comment by bilelmoussaoui** [comment]
> @bilelmoussaoui The build currently fails due to wrong metadata_license in appdata.yml
> 
> Afaik, it should be valid, according to the spdx code.

Metadata license shouldn't be a GPL. see here for the reasoning behind https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html

**Comment by daniel-wtd** [comment]
ouhhh, thats a tricky one, since it was originally gpl3. Rewriting the metadata_license to something else without the original maintainer? :/

**Comment by bilelmoussaoui** [comment]
I don't think @Alexander-Wilms would be against re-distributing the metainfo under the correct license.

File: com.shatteredpixel.shatteredpixeldungeon.yaml [hunk 1]
```diff
@@ -44,6 +44,6 @@ modules:
   - type: file
     path: com.shatteredpixel.shatteredpixeldungeon.appdata.xml
   - type: file
-    url: https://raw.githubusercontent.com/00-Evan/shattered-pixel-dungeon-gdx/master/android/res/drawable-xxxhdpi/ic_launcher.png
-    sha256: 3558bf1fa91c37e5cb48e91396c99b4fc3b37ffdb22381e6de9711548ef91890
+    url: https://raw.githubusercontent.com/00-Evan/shattered-pixel-dungeon/master/desktop/src/main/assets/icon_128.png
```
Comment (bilelmoussaoui) [inline com.shatteredpixel.shatteredpixeldungeon.yaml:47]: You need to fix the install destination as well, it's currently installed as a 48px icon.

Comment (daniel-wtd) [reply com.shatteredpixel.shatteredpixeldungeon.yaml:47]: yup, currently checking if the old dest just needs to be tweaked for 128x128 or something else will be the next issue ^^

**Comment by bilelmoussaoui** [comment]
Mind testing the latest build if everything works as expected? otherwise lgtm

thanks!

**Comment by daniel-wtd** [comment]
the very first tests seem fine, will play a bit and ping you tomorrow.

Other issues will be opened. After the merge of the one, #7 and #5 can be dropped. Hopefully, @whitel and @Lonsfor will conitnue to support the flatpak ;)

**Comment by Lonsfor** [comment]
i missed that the -gdx repo got deprecated but i'll try to keep up now

File: com.shatteredpixel.shatteredpixeldungeon.appdata.xml [hunk 2]
```diff
@@ -26,7 +26,7 @@
 		<p>Shattered Pixel Dungeon is open source software under the GPLv3 Licence. You can find the current source code here: https://github.com/00-Evan/shattered-pixel-dungeon</p>
 	</description>
 	<developer_name>Evan Debenham</developer_name>
-	<url type="homepage">https://www.shatteredpixel.com/</url>
+	<url type="homepage">https://shatteredpixel.com/</url>
 	<url type="bugtracker">https://github.com/00-Evan/shattered-pixel-dungeon-gdx/issues</url>
```
Comment (Lonsfor) [inline com.shatteredpixel.shatteredpixeldungeon.appdata.xml:30]: since repo got changed then this should be changed too

Comment (daniel-wtd) [reply com.shatteredpixel.shatteredpixeldungeon.appdata.xml]: thank you :) I missed that. updated

**Comment by daniel-wtd** [comment]
@bilelmoussaoui the tests look fine. no issues so far, no errors from flatpak.

**Comment by bilelmoussaoui** [comment]
thank you

**Pull request merged**